### PR TITLE
Document sensuctl delete

### LINKED ISSUE #1555: Document sensuctl delete command
Page: sensuctl reference

Reference: https://github.com/sensu/sensu-go/pull/3070

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

- Adds `sensuctl delete` to the sensuctl reference
- Updates the datastore reference to include the process for using `sensuctl delete` to disable the PostgreSQL event store

## Motivation and Context

Closes #1555

### CODE REVIEW

File: content/sensu-go/5.11/reference/datastore.md [hunk 1]
```diff
@@ -79,10 +77,21 @@ With the `PostgresConfig` resource definition saved to a file (for example: post
...
+
+To disable the PostgreSQL event store, use `sensuctl delete` with your `PostgresConfig` resource definition:
+
+{{< highlight shell >}}
+sensuctl delete --file postgres.yml
+{{< /highlight >}}
+
+When disabling the PostgreSQL event store, event data cuts over from PostgreSQL to etcd, resulting in a loss of recent event history.
+No restarts or Sensu backend configuration changes are required to disable the PostgreSQL event store.
+
```
Comment (apaskulin) [inline content/sensu-go/5.11/reference/datastore.md]: Does `sensuctl delete` require 100% match between all resource attributes or just name and namespace?

File: content/sensu-go/5.11/reference/datastore.md [hunk 2]
```diff
@@ -79,10 +77,21 @@ With the `PostgresConfig` resource definition saved to a file (for example: post
...
 To update your Sensu PostgreSQL configuration, repeat the `sensuctl create` process shown above.
 You can expect to see PostgreSQL status updates and error messages in the [Sensu backend logs][2] at the `warn` and `error` log levels, respectively.
 
+#### Disabling the PostgreSQL event store
+
+To disable the PostgreSQL event store, use `sensuctl delete` with your `PostgresConfig` resource definition:
+
+{{< highlight shell >}}
+sensuctl delete --file postgres.yml
+{{< /highlight >}}
```
Comment (apaskulin) [inline content/sensu-go/5.11/reference/datastore.md]: Does `sensuctl delete` support multi-namespace functionality similar to [`sensuctl create`](https://docs.sensu.io/sensu-go/5.10/sensuctl/reference/#creating-resources-across-namespaces)?

File: content/sensu-go/5.11/sensuctl/reference.md [hunk 3]
```diff
@@ -329,6 +330,23 @@ To create the `pagerduty` handler in the current session namespace:
...
+## Deleting resources
+
+The `sensuctl delete` command allows you to delete resources by reading from STDIN or a flag configured file (`-f`).
+The `delete` command accepts Sensu resource definitions in `wrapped-json` and `yaml` and uses the same [resources types][3] as `sensuctl create`.
+
+To delete all resources from `my-resources.yml` using `sensuctl delete`:
+
+{{< highlight shell >}}
+sensuctl delete --file my-resources.yml
+{{< /highlight >}}
```
Comment (apaskulin) [inline content/sensu-go/5.11/sensuctl/reference.md:345]: Does `sensuctl delete` support multi-namespace functionality similar to [`sensuctl create`](https://docs.sensu.io/sensu-go/5.10/sensuctl/reference/#creating-resources-across-namespaces)?

File: content/sensu-go/5.11/sensuctl/reference.md [hunk 4]
```diff
@@ -329,6 +330,23 @@ To create the `pagerduty` handler in the current session namespace:
 sensuctl create --file pagerduty.yml
 {{< /highlight >}}
 
+## Deleting resources
+
+The `sensuctl delete` command allows you to delete resources by reading from STDIN or a flag configured file (`-f`).
+The `delete` command accepts Sensu resource definitions in `wrapped-json` and `yaml` and uses the same [resources types][3] as `sensuctl create`.
```
Comment (apaskulin) [inline content/sensu-go/5.11/sensuctl/reference.md]: Does `sensuctl delete` require 100% match between all resource attributes or just name and namespace?

Comment (echlebek) [reply content/sensu-go/5.11/sensuctl/reference.md]: Just the name and namespace. However, the resource must be valid.

File: content/sensu-go/5.11/sensuctl/reference.md (hunk 3 above)
Comment (echlebek) [reply content/sensu-go/5.11/sensuctl/reference.md:345]: Good question. I think so, I need to look into this before confirming.

Comment (echlebek) [reply content/sensu-go/5.11/sensuctl/reference.md:345]: Confirmed!

**Review by echlebek — APPROVED** [review]

**Pull request merged**